presto-spi/src/main/java/com/facebook/presto/spi/block/SingleRowBlockWriter.java

```diff
@@ -27,16 +27,18 @@
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(SingleRowBlockWriter.class).instanceSize();
 
     private final BlockBuilder[] fieldBlockBuilders;
+    private final int numFields;
```

Comment:
Remove this, use `fieldBlockBuilders.length` to be consistent with `SingleRowBlock`